skipruntime-ts/core/src/internals/skipruntime_module.ts

```diff
@@ -1242,6 +1479,9 @@ class SkipRuntimeImpl implements SkipRuntime {
     }
   }
 
+  /**
+   * Close all the resource and shutdown the SkipService
+   */
   close(): void {
     const result = this.refs.skjson.runWithGC(() => {
       return this.refs.fromWasm.SkipRuntime_closeService();
```

Comment:
What's the behaviour if I call other runtime functions after `close`?  That would be a bug of course but I don't see any explicit logic about it -- would be good to at least make a note in this comment.